src/components/button.tsx

```diff
@@ -0,0 +1,26 @@
+import { Button as TamaguiButton, styled } from 'tamagui';
+
+export const Button = styled(TamaguiButton, {
+  pressStyle: {
+    opacity: 0.9,
+  },
+
+  variants: {
+    mode: {
+      primary: (_, { tokens }) => ({
+        height: 56,
+        borderRadius: '$11',
```

Comment:
Let's invent `round` token:

```suggestion
        borderRadius: '$round',
```

`tamagui.config.ts`
```
...
radius: {
    ...radius,
    3: 8,
    'round': 999,
  },
...
```